client/src/components/Behodler/Swap/TradingBox3/TokenSelector/Menu.tsx

```diff
@@ -120,18 +118,19 @@ export default function Menu(props: props) {
     );
     const indexOfWeth = tokenList.findIndex((item) => item.name.toLowerCase().indexOf("weth") !== -1);
     const indexOfScarcityAddress = tokenList.findIndex((item) => item.name.toLowerCase().indexOf("scarcity") !== -1);
-    const behodler2Weth = props.weth10Address;
+    const weth10Address = tokenList[indexOfWeth].address;
+    const scxAddress = tokenList[indexOfScarcityAddress].address;
```

Comment:
weth10 and SCX addresses were hard-coded in the parent component and passed as props, now proper addresses will be used depending on the active network